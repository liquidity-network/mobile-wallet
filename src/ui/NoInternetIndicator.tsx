import React, { PureComponent } from 'react'
import { View } from 'react-native'
import Feather from 'react-native-vector-icons/Feather'

import { BG_COLOR, RED, statusBarHeight } from 'globalStyles'

export default class NoInternetIndicator extends PureComponent {
  render() {
    return (
      <View style={styles.container}>
        <Feather name="wifi-off" size={22} color={RED} />
      </View>
    )
  }
}

const styles: any = {
  container: {
    position: 'absolute',
    top: statusBarHeight + 8,
    right: 10,
    width: 40,
    height: 40,
    backgroundColor: BG_COLOR,
    borderRadius: 20,
    borderWidth: 1,
    borderColor: RED,
    alignItems: 'center',
    justifyContent: 'center',
  },
}
