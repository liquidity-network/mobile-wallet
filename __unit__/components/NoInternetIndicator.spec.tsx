import React from 'react'
import { create } from 'react-test-renderer'

import NoInternetIndicator from 'ui/NoInternetIndicator'

describe('<NoInternetIndicator> component', () => {
  it('renders correctly', () => {
    const tree = create(<NoInternetIndicator />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
