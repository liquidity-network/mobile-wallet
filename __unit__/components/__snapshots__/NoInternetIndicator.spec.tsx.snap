// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoInternetIndicator> component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0f022b",
      "borderColor": "#d11b1b",
      "borderRadius": 20,
      "borderWidth": 1,
      "height": 40,
      "justifyContent": "center",
      "position": "absolute",
      "right": 10,
      "top": 28,
      "width": 40,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "#d11b1b",
          "fontSize": 22,
        },
        undefined,
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ïˆ…
  </Text>
</View>
`;
